images/linux/scripts/installers/containers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  containers.sh
```

Comment:
@phozzy since this is not applicable to 16.04 can you move it under the 18.04 folder please? `images/linux/scripts/installers/1804`

Reply:
@alepauly Done.